packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -40,16 +40,17 @@ class SubscriptionListener {
 
 enum SocketConnectionState { notConnected, connecting, connected }
 
-class SocketClientConfig {
-  const SocketClientConfig({
+class customConnect {
```

Comment:
Typo?

Reply:
Ops! Sorry about that and thanks for the catch!